cmd/crc/cmd/start.go

```diff
@@ -55,7 +55,7 @@ func runStart(arguments []string) {
 
 	commandResult, err := machine.Start(startConfig)
 	if err != nil {
-		errors.Exit(1)
+		logging.Fatal(err)
```

Comment:
why we want this change, does it show the error message and have exit code non-zero? cc @cfergeau

Reply:
This just exits, and doesnot print the err message, and user does not have any clue as to why it exited, and yes it throws exit code 1

Reply:
Wouldn't it then be a good idea to modify the Exit method to also accept a message?

Reply:
Yes, we already have `ExitWithMessage`, will use that!

Reply:
logging exiting when fatal is used sounds very unintuitive. I would not expect logging to exit (feels like a side-effect)... even errors performing an exit is weird, but at least the function name makes you expect it to happen.